src/query/xasl.h

```diff
@@ -1066,7 +1057,10 @@ struct xasl_node
   int dbval_cnt;		/* number of host variables in this XASL */
   bool iscan_oid_order;
 
-  int max_iterations;		/* Number of maximum iterations (used during run-time for recursive CTE) */
+#if defined (CS_MODE) || defined (SA_MODE)
```

Comment:
not CS_MODE but SA_MODE?

Reply:
No, they are only used in query optimizer.